platforms/r3-corda-ent/charts/auth/templates/deployment.yaml

```diff
@@ -128,9 +142,43 @@ spec:
           command: ["/bin/sh", "-c"]
           args:
           - |-
-            echo "CENM: starting CENM Auth service ..."
-            # [TODO] : initial-user-password to be fetched from vault 
-            java -jar bin/accounts-application.jar --config-file authservice.conf --initial-user-name admin --initial-user-password p4ssWord --keep-running --verbose
+            mkdir -p {{ .Values.config.volume.baseDir }}/config;
+
+            install {{ .Values.config.volume.baseDir }}/authservice.conf {{ .Values.config.volume.baseDir }}/config;
```

Comment:
what does this line do?

Reply:
it copies the read only file to another directory with read write permissions without changing the indentation